### ISSUE TITLE
pull-kubernetes-e2e-kops-aws failing

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/kubernetes#73444]
**Which jobs are failing**: pull-kubernetes-e2e-kops-aws

**Which test(s) are failing**: Up

**Since when has it been failing**:
Last pass @ 2019-01-28 05:43 PST https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/73295/pull-kubernetes-e2e-kops-aws/120805
First failure @ 2019-01-28 05:47 PST
https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/73318/pull-kubernetes-e2e-kops-aws/120806

**Testgrid link**: https://testgrid.k8s.io/presubmits-kubernetes-blocking#pull-kubernetes-e2e-kops-aws

**Reason for failure**: TBD

**Anything else we need to know**: Would have reported this sooner but was obscured by everything else failing due to https://github.com/kubernetes/test-infra/issues/11001

**Comment by spiffxp** [comment]
/sig cluster-lifecycle
because kops
/sig aws
because this is running under a sig-aws managed account AFAIK

**Comment by spiffxp** [comment]
/priority critical-urgent
this is merge blocking

**Comment by spiffxp** [comment]
Looks like this isn't just the presubmit: https://testgrid.k8s.io/sig-cluster-lifecycle-kops#kops-aws

- Last pass started @ 2019-01-28 05:54 PST https://gubernator.k8s.io/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kops-aws/38326
- First fail started @ 2019-01-28 06:35 PST https://gubernator.k8s.io/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kops-aws/38327

**Comment by spiffxp** [comment]
[triage link: "waiting for ready nodes timed out"](https://storage.googleapis.com/k8s-gubernator/triage/index.html?pr=1&text=waiting%20for%20ready%20nodes%20timed%20out)

**Comment by spiffxp** [comment]
The CI jobs going from green to red without the underlying kubernetes commit or kubernetes test-infra commit changing make me feel like this is an environment issue, I'm not sure if the version of kops being used changed?

**Comment by justinsb** [comment]
I poked around the AWS account, we're getting this error message from the AWS ASGs:

`Launching a new EC2 instance. Status Reason: This account is currently blocked and not recognized as a valid account. Please contact aws-verification@amazon.com if you have questions. Launching EC2 instance failed.`

**Comment by spiffxp** [comment]
/assign @shyamjvs 
I know you've been doing test-infra work related to presets for AWS credentials, maybe related?

**Comment by shyamjvs** [comment]
@BenTheElder Have we changed sth around this recently? I remember this was fixed few days ago when it broke.

**Comment by shyamjvs** [comment]
Sorry, also @krzyzacy

**Comment by justinsb** [comment]
To be clear though, kops is creating the ASGs, security groups & networking resources, so it isn't a normal "wrong password" problem.  It looks like AWS has maybe decided our account can't create instances?  Like our account has been flagged?

**Comment by shyamjvs** [comment]
Thanks for the lead. I'll try to follow up on this internally.

**Comment by BenTheElder** [comment]
I've not touched anything, I only reverted the _naming_ of the presets in the config, and I don't have access to any of the AWS accounts. I don't believe @krzyzacy has either. Looks like an issue with the account.

**Comment by shyamjvs** [comment]
@justinsb Do you know who/how the account was created?

**Comment by justinsb** [comment]
It's a CNCF-owned/administered account - I think @idvoretskyi is the contact

**Comment by krzyzacy** [comment]
I think I can also poke, 1min

**Comment by shyamjvs** [comment]
I suggest temporarily making the presubmit non-blocking, until this fixed. WDYT?

**Comment by krzyzacy** [comment]
sgtm

**Comment by shyamjvs** [comment]
I'll send out a PR making the change.

**Comment by shyamjvs** [comment]
![screen shot 2019-01-28 at 7 50 26 pm](https://user-images.githubusercontent.com/4333971/51883056-29eb3f80-2336-11e9-8dd1-20826d3791fd.png)

^ I think this is the reason for the block.

**Comment by shyamjvs** [comment]
The payments haven't been made from Oct 2018. @idvoretskyi Seems like there's a card registered on your name. Can we go ahead and make those pending payments?

**Comment by shyamjvs** [comment]
> I'll send out a PR making the change.

Sent https://github.com/kubernetes/test-infra/pull/11018

**Comment by idvoretskyi** [comment]
@d-nishi @arun-gupta please, take a look.

**Comment by d-nishi** [comment]
@shyamjvs @idvoretskyi @justinsb - I added credits to the CNCF account in Oct 2018 and later modified them to cover all regions. Have we run out of the credits again?

**Comment by d-nishi** [comment]
sounds like the credits are all done and @arun-gupta is trying to refresh them on the CNCF account with @shyamjvs

**Comment by zetaab** [comment]
is there any news when this issue is solved?

**Comment by shyamjvs** [comment]
We're expecting to get this resolved early next week.

**Comment by zetaab** [comment]
@shyamjvs any news?

**Comment by dims** [comment]
@zetaab, I believe @idvoretskyi @arun-gupta are on point, we will get an ETA hopefully in a day or so.

**Comment by shyamjvs** [comment]
A request for $19K credit has been approved. This will be applied before this month automatically to the account. Seems like we can start using EC2 now.

/cc @arun-gupta - for confirmation

**Comment by shyamjvs** [comment]
I'm trying to debug issues with the presubmit under PR https://github.com/kubernetes/kubernetes/pull/73797. Currently I'm blocked on getting access to the account. I've requested @idvoretskyi for it and he said he'll respond within a day.

**Comment by idvoretskyi** [comment]
@shyamjvs Shyam, the account is still in a suspended status; I'm not able to make any changes yet. Updated the mail thread with @arun-gupta about this issue.

**Comment by idvoretskyi** [comment]
Status update: the billing issue is still in place, we are currently working on it with AWS.

**Comment by zetaab** [comment]
@idvoretskyi is there possibility to somehow solve this issue? We have been now waiting for this over 2 weeks. All development in case of KOPS is on hold. In my opinion this is not very good situation, of course we could just skip aws e2e tests, but then there is possibility that we end up to production problems. There is issue like CVE problems, that we should somehow merge to releases but we cannot tests those.

**Comment by idvoretskyi** [comment]
@zetaab Shyam @shyamjvs is able to provide more details on this issue.

**Comment by idvoretskyi** [comment]
Status update: we are actively working with AWS to resolve the issue. It is currently unsolved, but hopefully, we'll have more details later today.

**Comment by shyamjvs** [comment]
Sorry for delay.. had to pull a bunch of strings internally. We discussed this in a meeting today and have clear action items currently being worked upon, to get account activated and payments approved. The situation got complicated because of missing multiple payments and neglecting communication for multiple months.

Expect another update by tomorrow morning PST.

**Comment by idvoretskyi** [comment]
Thanks @shyamjvs

**Comment by shyamjvs** [comment]
Update as promised - The account has been unsuspended by fraud/verification team and is no longer under risk of closure. It takes 24 hours to activate. Meanwhile, the support team is working on applying new credits retroactively to past pending payments. Once that's approved (ETA tomorrow), account will be usable again. Will keep this issue updated.

**Comment by idvoretskyi** [comment]
Great, thanks @shyamjvs!

**Comment by shyamjvs** [comment]
Good news - Just got the kops-aws presubmit running (see https://github.com/kubernetes/kubernetes/pull/73797#issuecomment-463827056). Also you can see the kops CI jobs recovering.. https://testgrid.k8s.io/google-aws#kops-aws. That said, the payments issue is being fixed in the background now.

There are still couple e2e tests that are failing with [sig-storage] (seems like regression that entered during this period). We'll need to fix those before turning on the presubmit, but that's a separate issue (let me file issue for it).

**Comment by shyamjvs** [comment]
https://github.com/kubernetes/test-infra/pull/11353 should add back the kops-aws presubmit. Kops-aws CI jobs look pretty good too - https://testgrid.k8s.io/google-aws#kops-aws

**Comment by shyamjvs** [comment]
We're not enabling back the presubmit per discussion under https://github.com/kubernetes/test-infra/pull/11353. Closing this issue; reopen if apt.

/close
